src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting_NetFx/FormattingLanguageServerTestBase.cs

```diff
@@ -65,5 +66,11 @@ public Task<TextEdit[]> GetHtmlOnTypeFormattingEditsAsync(DocumentContext docume
         {
             throw new NotImplementedException();
         }
+
+        public bool TryGetOnTypeFormattingTriggerKind(RazorCodeDocument codeDocument, int hostDocumentIndex, string triggerCharacter, out RazorLanguageKind triggerCharacterKind)
+        {
+            triggerCharacterKind = languageKind ?? RazorLanguageKind.CSharp;
```

Comment:
It feels a bit strange to me that this would set `RazorLanguageKind` to a real value in the case that it returns false?

```suggestion
            triggerCharacterKind = languageKind ?? 0;
```

Reply:
Remember this morning how I said there were two tests that I was on the fence between updating, or deleting? Guess which two tests are the only ones to call this method 😁